package/kernel/linux/modules/video.mk

```diff
@@ -503,6 +503,23 @@ endef
 
 $(eval $(call KernelPackage,drm-i915))
 
+define KernelPackage/drm-ivpu
+  SUBMENU:=$(VIDEO_MENU)
+  TITLE:=Intel VPU DRM support
+  DEPENDS:=@TARGET_x86 +ivpu-firmware
+  KCONFIG:=CONFIG_DRM_ACCEL_IVPU \
+       CONFIG_DRM_ACCEL_HABANALABS=n \
+       CONFIG_DRM_ACCEL_QAIC=n \
+       CONFIG_DRM_ACCEL=y
```

Comment:
Considering that DRM_ACCEL guards both multiple drivers, it should be its own kmod (It can even be hidden as its boolean)

Reply:
Since DRM_ACCEL is a gatekeeping Kconfig option, enabling it is required for DRM_ACCEL_IVPU and other related kernel modules (qaic, habanalabs). Is it OK to create a dummy package and selected by other dependent package?

```mk
define KernelPackage/drm-accel
  SUBMENU:=Graphics Drivers
  TITLE:=Compute Acceleration Framework
  KCONFIG:=CONFIG_DRM_ACCEL
  FILES:=
  AUTOLOAD:=
endef

define KernelPackage/drm-accel/description
  Enables the Compute Acceleration Framework for AI and ML accelerators.
endef

$(eval $(call KernelPackage,drm-accel))
```